Remove .data subset1 for fixathon

pbelevich opened the pull request:
To test:
* `benchmarks/fastrnns/bench.py` run `python -m fastrnns.bench`
* `benchmarks/fastrnns/profile.py` run `python -m fastrnns.profile`
* `torch/utils/tensorboard/_convert_np.py` run TODO
* `test/test_numba_integration.py` run `python test/test_numba_integration.py`

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#33880 Remove .data subset1 for fixathon**

pbelevich committed:
Remove .data subset1 for fixathon

[ghstack-poisoned]

pbelevich committed:
Update on "Remove .data subset1 for fixathon"




[ghstack-poisoned]

SsnL commented on torch/utils/tensorboard/_convert_np.py:
```diff
@@ -32,7 +32,7 @@ def make_np(x):
 def _prepare_pytorch(x):
     if isinstance(x, torch.autograd.Variable):
-        x = x.data
+        x = x.detach()
```
`Variable` and `Tensor` are the same thing now... The better fix is probably remove `if`, and change next line to `x = x.detach().cpu().numpy()`.

pbelevich committed:
Update on "Remove .data subset1 for fixathon"

To test:
* `benchmarks/fastrnns/bench.py` run `python -m fastrnns.bench`
* `benchmarks/fastrnns/profile.py` run `python -m fastrnns.profile`
* `torch/utils/tensorboard/_convert_np.py` run TODO

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#33880 Remove .data subset1 for fixathon**

[ghstack-poisoned]

pbelevich committed:
Update on "Remove .data subset1 for fixathon"

To test:
* `benchmarks/fastrnns/bench.py` run `python -m fastrnns.bench`
* `benchmarks/fastrnns/profile.py` run `python -m fastrnns.profile`
* `torch/utils/tensorboard/_convert_np.py` run TODO

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#33880 Remove .data subset1 for fixathon**

[ghstack-poisoned]
